ui/src/app.js

```diff
@@ -135,6 +138,8 @@ function processData (data) {
   if (data.operatorInfo) setOperatorInfo(data.operatorInfo)
   if (data.hardLimit) setHardLimit(data.hardLimit)
   if (data.cryptomatModel) setCryptomatModel(data.cryptomatModel)
+  if (data.areThereAvailableCoupons !== undefined) setAvailableCoupons(data.areThereAvailableCoupons)
```

Comment:
can this be like this instead? `if (data.areThereAvailableCoupons) setAvailableCoupons(data.areThereAvailableCoupons)`

Reply:
it is a boolean, `if (data.areThereAvailableCoupons)` won't distinguish between undefined and false.